src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -569,24 +578,31 @@ private void replaceEC2Servers() {
 
```

Comment:
(Couldn't ref this line, so adding code ref below):

Is it still true that the graph build only instance will self-terminate in the user data? Also, should this be the end of the job or should it wait for recreateGraphBuildImageJob?

https://github.com/ibi-group/datatools-server/blob/06138de0bfc3abd6afa86c74e597a2c8fe70c6b6/src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java#L569-L574

Reply:
It won't automatically shut down, I will update the comment.